src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -264,15 +308,30 @@ const ModalCardDetails = ({
               ))}
             </div>
             <br />
-            <Subheader>Checkout details</Subheader>
+            <Subheader>{setDetailsText(purchaseType, amount)}</Subheader>
+
             <span>
               {' '}
-              {purchaseTypePhrase(false, purchaseType)} of{' '}
+              {purchaseTypeMessage(purchaseType, amount)} of{' '}
               <b>
                 ${amount} {numberOfMealsText}
               </b>{' '}
               to {sellerName}{' '}
             </span>
+
+            {licData.first_name !== '' && (
+              <span>
+                <br />
+                <br />
+                {t('modalPayment.modalCardDetails.message.lic_name')}
+                <b>
+                  {`${licData.first_name} ${
+                    licData.middle_initial.length > 0 &&
+                    licData.middle_initial.substring(0, 1).toUpperCase()
+                  }. ${licData.last_name}`}
```

Comment:
Two bugs:
1) If user does not enter middle name, it will display false
2) The full stop is outside of the braces of the middle name so if it's just first name and last name, it will add a full stop in between e.g. Alice . Luo 

```suggestion
                  {`${licData.first_name} ${
                    licData.middle_initial.length > 0 ?
                    licData.middle_initial.substring(0, 1).toUpperCase() + '. ' : ''
                  }${licData.last_name}`}